BREAKING.md

```diff
@@ -50,6 +50,7 @@ Previously, `DataObject` would perform undocumented special handling for request
 
 -   [Ability to enable grouped batching](#Ability-to-enable-grouped-batching)
 -   [Testing support for LTS is moved from 0.45 to 1.3.4](#Testing-support-for-LTS-is-moved-from-0.45-to-1.3.4)
+-   [MockLogger is now internal](#mocklogger-is-now-internal)
```

Comment:
i don't think we use breaking.md anymore, you should add a changeset

Reply:
Yeah, this is a pretty old PR. It'll be simpler if applied once MockLogger is not public API.